test/services/GoodsItemCachingData.scala

```diff
@@ -31,9 +31,9 @@ trait GoodsItemCachingData {
   def decimalString(): String = Random.nextDouble().toString
 
   def createPackageInformation(): PackageInformation = PackageInformation(
-    Some(createRandomAlphanumericString(2)),
-    Some(Random.nextInt(20)),
-    shippingMarks = Some(createRandomAlphanumericString(150))
+    createRandomAlphanumericString(2),
+    Random.nextInt(20),
```

Comment:
Here we could have 0 which would not pass validation - are we care for it?

Reply:
So could Some(0) to be fair.   Have fixed it in case it breaks a test that validates